LocalPackages/SyncUI/Sources/SyncUI/Views/DeviceConnectedView.swift

```diff
@@ -97,6 +105,36 @@ public struct DeviceConnectedView: View {
         .padding(.bottom)
     }
 
+    @ViewBuilder
+    func options() -> some View {
+        VStack {
+            Spacer(minLength: 71)
+            Text(UserText.options.uppercased())
+                .font(.system(size: 12))
```

Comment:
I know this wasn't in the code already, but could you go through and replace any calls to .font() with the appropriate modifier from DesignResourcesKit please? For instance, in Figma: 

<img width="241" alt="Screenshot 2023-10-09 at 19 15 58" src="https://github.com/duckduckgo/iOS/assets/81197/2c219d4e-553d-4edc-8af8-eb62acd89501">

This would be:

`daxBodyBold()`